build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactmetrics/ArtifactMetricsPlugin.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package aws.sdk.kotlin.gradle.plugins.artifactmetrics
+
+import org.gradle.api.GradleException
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.TaskProvider
+import org.gradle.jvm.tasks.Jar
+import org.gradle.kotlin.dsl.register
+import org.gradle.kotlin.dsl.withType
+
+private const val TASK_GROUP = "Verification"
```

Comment:
nit: Tasks in this group collect *and* analyze/verify metrics, so I don't think "Verification" is accurate

Reply:
How about "Metrics"? It's the common denominator between all the functionality in the plugin and "Artifact Metrics" seems too specific

Reply:
Sure that sounds good to me. It's under the artifactmetrics namespace after all